source/agora/common/Config.d

```diff
@@ -164,7 +180,12 @@ public GetoptResult parseCommandLine (ref CommandLine cmdline, string[] args)
 
         "config-check",
             "Check the state of the config and exit",
-            &cmdline.config_check);
+            &cmdline.config_check,
+
+        "override|O",
+            "Override a config file value",
```

Comment:
It would be nice to show an example how it's actually done, in this help message. is it just `-O validator=false` or `-O node:validator=false`, etc?